.github/check-registry.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+PORT=9991
+PEERS=(
```

Comment:
I hope no one will change private keys in docker-compose.yml and peer ids won't change 🙈